client/partials/list/th-list.html

```diff
@@ -1,35 +1,33 @@
 <!--Detailed table feeds-->
-<div id="th-list" class="row feeds">
+<div id="th-list" class="feeds">
     <div class="col-xs-12 th-list" ng-repeat="article in articles">
-        <div class="row">
-            <div class="panel panel-default app-panel fade-in-anim">
-                <div class="panel-body">
-                    <img ng-show="article.img" ng-src="{{article.img}}" class="img-responsive" alt="Test img">
-                    <div class="content">
-                        <div class="title">
-                            <p>
-                                <strong>
-                                    <span ng-bind="article.title"></span>
-                                </strong>
-                            </p>
-                        </div>
-                        <div class="description">
-                            <p>
-                                <span ng-bind="article.content"></span>
-                            </p>
-                        </div>
-                        <div class="footer">
-                            <div class="share">
-                                <span class="my-hidden">
+        <div class="panel panel-default app-panel fade-in-anim">
+            <div class="panel-body">
+                <img ng-show="article.img" ng-src="{{article.img}}" class="img-responsive" alt="Test img">
+                <div class="content">
+                    <div class="title">
```

Comment:
Better style class `title` and not spam dom tree with `p, storng` and `span` element